File: Assignment 6/Activity 3 .js
// This program converts the unit miles in to yards, feet, and inches.
//
// Reference:
//  https://www.mathsisfun.com/measure/us-standard-length.html
//  https://en.wikibooks.org/wiki/JavaScript

function main() {
    var miles;
    var yards;
    var feet;
    var inches;
    
    miles = getmiles();
    yards = calculateyards(miles);
    feet = calculatefeet(miles);
    inches = calculateinches(miles);
    displayResult(yards, feet, inches);
}

function calculatefeet(miles) {
    var feet;
    
    feet = miles * 5280;
    
    return feet;
}

function calculateinches(miles) {
    var inches;
    
    inches = miles * 63360;
    
    return inches;
}

function calculateyards(miles) {
    var yards;
    
    yards = miles * 1760;
    
    return yards;
}

function displayResult(yards, feet, inches) {
    window.alert(yards.toString() + " yards " + feet + " feet " + inches + " inches ");
}

function getmiles() {
    var miles;
    
    window.alert("Enter a value for miles");
    miles = window.prompt('Enter a value for miles');
    
    return miles;
}

File: Assignment 6/Activity 3.py
# This program converts the unit miles in to yards, feet, and inches.
# Reference: https://www.mathsisfun.com/measure/us-standard-length.html
# https://en.wikibooks.org/wiki/JavaScript

def calculate_feet(miles):
    feet = miles * 5280    
    return feet


def calculate_inches(miles):
    inches = miles * 63360
    return inches


def calculate_yards(miles):
    yards = miles * 1760   
    return yards


def display_result(yards, feet, inches):
    print(str(yards) + " yards " + 
    str(feet) + " feet " + str(inches) + " inches ")


def get_miles():
    print("Enter a value for miles")
    miles = float(input())
    return miles

# main function starts here
def main():
    miles = get_miles()
    yards = calculate_yards(miles)
    feet = calculate_feet(miles)
    inches = calculate_inches(miles)
    display_result(yards, feet, inches)
    
main()
